src/components/Selection.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import Parse from "parse";
+
+
+export function Selection() {
+  const [selection, setSelection] = useState();
+
+  useEffect(() => {
+    const Idea = Parse.Object.extend("Idea");
+    const query = new Parse.Query(Idea);
+
+    const ideas = query.find().then((ideas) => {
+      console.log(ideas);
+      setSelection(ideas);
+    });
+  }, []);
+
+  if (!selection) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <>
+       <br />
+      <h1>Selection</h1>
+      <br />
+      <br />
+      <ul>
+      {selection.map((idea, i) => (
+       <li key={i}><img src={ idea.get("image").url()} alt="illustration expressing the idea"  /><br/><b>{idea.get("title")}</b><br/>{idea.get("description")}<hr/></li>
+      ))}
+      </ul>
+    </>
+  );
+}
```

Comment:
There are differing opinions on having 2 return statements.
I know that Sara dislikes them, however. I can read this just fine, but there is an argument for readability supporting 1 return statement.

You can move the if down into the main return block with a ternary statement (a sexy version of "if-else").

Reply:
I realize how this reads, so just be absolutely clear:
I am **not** saying that I can read this fine, but Sara cannot.
I mention Sara's opinion because I respect her, and would like to defer to her on this matter because I think she has **proper reasons** for disliking 2 return statements, whereas I base my own _lack-of-opinion-tilting-to-the-side-of-dislike_ merely on a feeling.

In either case, it's not as much an error on your part, as something I want you to be aware of.